docs/getting-started/android/index.md

```diff
@@ -15,7 +15,7 @@ A guide on how to integrate Warp into your project.
 Warp is used together with a brand theme and should be installed for a specific flavor of the code. Currently only Finn and Tori are supported.
 
 ```gradle
```

Comment:
We get warnings that "gradle" is not a supported language... is it possible to change to something else?
https://shiki.style/languages
...or fix it in some other way? 😄 
https://shiki.style/guide/load-lang

Reply:
Right, i should use groovy here.